schedule/index.html

```diff
@@ -123,6 +123,9 @@
                 <li><a href="#2016-03-20-exhibition">Exhibition</a></li>
               </ul>
           </ul>
+          <form class="pull-right" style="padding:1%">
```

Comment:
@joerx Can you please , why to submit PR for the markup change in 2016.fossasia.org to open-event-scraper.git explain it ?